molgenis-core-ui/src/main/resources/templates/view-staticcontent-edit.ftl

```diff
@@ -1,40 +1,42 @@
+<#include "resource-macros.ftl">
 <#include "molgenis-header.ftl">
 <#include "molgenis-footer.ftl">
-<#assign css=[]>
-<#assign js=["ckeditor/ckeditor.js"]>
-<@header css js/>
+
+<script src="<@resource_href "/js/ckeditor/ckeditor.js"/>"></script>
+<@header css=[] js=[] version=2/>
+
 <div class="row">
-    <div class="col-md-12">
-        <form id="contentForm" method="post" role="form">
+  <div class="col-12">
+    <form id="contentForm" method="post" role="form">
 
-            <div class="form-group">
-                <div class="col-md-8 col-md-offset-2">
+      <div class="form-group">
+        <div class="col-8 offset-md-2">
                 <#if content?has_content>
-                    <textarea id="elm1" name="content" form="contentForm"
-                              rows="15">${content} <#if succes?has_content>${succes?html}</#if></textarea>
+                  <textarea id="elm1" name="content" form="contentForm"
+                            rows="15">${content} <#if succes?has_content>${succes?html}</#if></textarea>
                 <#else>
                     <textarea id="elm1" name="content" form="contentForm" rows="15"></textarea>
                 </#if>
-                </div>
-            </div>
+        </div>
+      </div>
 
-            <div class="form-group">
-                <div class="col-md-8 col-md-offset-2">
-                    <div class="btn-group pull-right">
-                        <a id="cancelBtn" href="${context_url?html}" class="btn btn-default">Close</a>
-                        <button id="submitBtn" type="submit" class="btn btn-default">Save</button>
-                    </div>
-                </div>
-            </div>
+      <div class="form-group">
+        <div class="col-8 offset-md-2">
+          <div class="btn-group float-right">
+            <a id="cancelBtn" href="${context_url?html}" class="btn btn-secondary">Close</a>
+            <button id="submitBtn" type="submit" class="btn btn-secondary">Save</button>
```

Comment:
i suggest primary as submit is the primary action ( i see it used to be default, but still)